jobs/delorean/jenkinsfiles/branch/ga/Jenkinsfile

```diff
@@ -61,6 +61,60 @@ GHPullRequest ghFindOrCreatePullRequest(GHRepository repo, String head, String b
     return pr
 }
 
+def hasJiraUpdateIssue(credentials, product, productVersion) {
+    def query = "project = INTLY AND status = Open AND labels=product-update AND labels=${product} AND labels='${product}:${productVersion}'"
+    def body = """{
+        "jql": "${query}"
+    }"""
+    def url = 'https://issues.jboss.org/rest/api/2/search'
+    def res = httpRequest quiet: true, authentication: credentials, consoleLogResponseBody: true, contentType: 'APPLICATION_JSON', httpMode: 'POST', requestBody: body, responseHandle: 'NONE', timeout: 60, url: url, validResponseCodes: '200'
+    def data = readJSON text: "${res.content}"
+    
+    def found = res.status == 200 && data.total > 0
+    def key = ''
+    if (found) {
+        key = data.issues[0].key
+    }
+
+    return [found, key]
+}
+
+def createJiraIssue(credentials, gitUrl, product, version) {
+    def body = """{
+        "fields": {
+            "project": {
+                "id": "12321620",
+                "key": "INTLY",
+                "name": "Integreatly"
+            },
+            "summary": "New version available for ${product}",
```

Comment:
Can we add the version into here now? Maybe make the summary more like the ones that had been manually created "Update to Fuse 7.3" https://issues.jboss.org/browse/INTLY-1744